x/projects/keeper/project_test.go

```diff
@@ -242,41 +304,58 @@ func SetPolicyTest(t *testing.T, testAdminPolicy bool) {
 		setPolicySuccess     bool
 	}{
 		{
-			"valid policy (admin account)", adminAcc.Addr.String(),
+			"valid policy (admin account)", adminAcc.Addr.String(), projectID, uint64(1),
 			[]types.ChainPolicy{{ChainId: spec.Index, Apis: []string{spec.Apis[0].Name}}},
 			100, 10, 3, true, true,
 		},
 
 		{
-			"valid policy (subscription account)", subAccount.Addr.String(),
+			"valid policy (subscription account)", subAccount.Addr.String(), projectID, uint64(1),
 			[]types.ChainPolicy{{ChainId: spec.Index, Apis: []string{spec.Apis[0].Name}}},
 			100, 10, 3, true, true,
 		},
 
 		{
-			"bad creator (developer account -- not admin)", developerAcc.Addr.String(),
+			"bad creator (developer account -- not admin)", developerAcc.Addr.String(), projectID, uint64(1),
+			[]types.ChainPolicy{{ChainId: spec.Index, Apis: []string{spec.Apis[0].Name}}},
+			100, 10, 3, true, false,
+		},
+
+		{
+			"bad projectID (doesn't exist)", developerAcc.Addr.String(), "fakeProjectId", uint64(1),
 			[]types.ChainPolicy{{ChainId: spec.Index, Apis: []string{spec.Apis[0].Name}}},
 			100, 10, 3, true, false,
 		},
 
 		{
-			"bad chainID (doesn't exist)", subAccount.Addr.String(),
+			"invalid geolocation (0)", developerAcc.Addr.String(), "fakeProjectId", uint64(0),
```

Comment:
not sure I understand your comment

Reply:
oops... regex syntax to say: substitute the **"fakeProjectId"** with **projectID**.
otherwise the test will fail as planned because of bad projectID and not because of invalid geolocation as needed.